packages/core/admin/admin/src/translations/ko.json

```
@@ -127,22 +128,23 @@
   "Settings.permissions.users.listview.header.subtitle": "{number, plural, =0 {# 명} one {# 명} other {# 명}}의 사용자를 찾았습니다.",
   "Settings.permissions.users.listview.header.title": "사용자",
   "Settings.permissions.users.tabs.label": "Tabs Permissions",
-  "Settings.profile.form.notify.data.loaded": "Your profile data has been loaded",
+  "Settings.profile.form.notify.data.loaded": "사용자 프로필 정보를 불러왔습니다.",
   "Settings.profile.form.section.experience.clear.select": "Clear the interface language selected",
   "Settings.profile.form.section.experience.interfaceLanguage": "인터페이스 언어",
   "Settings.profile.form.section.experience.interfaceLanguage.hint": "선택한 언어로 인터페이스의 언어가 변경됩니다.",
-  "Settings.profile.form.section.experience.title": "Experience",
+  "Settings.profile.form.section.experience.interfaceLanguageHelp": "선택하면 이 계정에서만 인터페이스 언어가 변경됩니다. 팀에서 다른 언어를 사용할 수 있도록 하려면 이 {문서}를 참조해주세요.",
```

Here `{documentation}` is a variable name and should not be translated
```suggestion
  "Settings.profile.form.section.experience.interfaceLanguageHelp": "선택하면 이 계정에서만 인터페이스 언어가 변경됩니다. 팀에서 다른 언어를 사용할 수 있도록 하려면 이 {documentation}를 참조해주세요.",
```

---

Good eyes, Thanks!